src/compas_timber/connections/t_butt.py

```diff
@@ -36,6 +37,18 @@ class TButtJoint(ButtJoint):
     def __init__(self, main_beam=None, cross_beam=None, mill_depth=0, **kwargs):
         super(TButtJoint, self).__init__(main_beam, cross_beam, mill_depth, **kwargs)
 
+    def get_cutting_plane(self):
+        assert self.main_beam and self.cross_beam  # should never happen
```

Comment:
I use these here and there mostly to calm Pylance down. with the `#type: ()->..` comments it would complain about stuff which could be `None`. anyways it's basically a debug check. if beams are `None` there, it's not a user error.